core/dbt/node_types.py

```diff
@@ -26,14 +26,14 @@ class NodeType(StrEnum):
     Seed = "seed"
     # TODO: rm?
     RPCCall = "rpc"
-    SqlOperation = "sql operation"
+    SqlOperation = "sqloperation"
     Documentation = "doc"
     Source = "source"
     Macro = "macro"
     Exposure = "exposure"
     Metric = "metric"
     Group = "group"
-    SemanticModel = "semantic model"
+    SemanticModel = "semanticmodel"
```

Comment:
This is not in scope of the current fix; it just annoys me that there's a space in the `unique_id`. I can kick out and open as a separate PR

Reply:
Opened as separate issue: https://github.com/dbt-labs/dbt-core/issues/7841

Will move commit to separate PR